content/ui/home.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div id="browsingTime">
+        <h1>Ping kids mode!</h1>
```

Comment:
on logging in new tab page should be shown and the browser extension icon should show that the kids mode is enabled along with a tooltip that kids mode is enabled